_posts/2016-02-01-melbourne.html

```diff
@@ -142,25 +142,25 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-12 text-center">
-                <h1>Join On In</h1>
```

Comment:
Would recommend softer language than 'apply' - I understand where you're coming from, but makes it sound non-inclusive (a lot of people will figure they won't "make the cut", whatever that means, as soon as there's an application process).